### DISCUSSION TITLE
ΓΕ2-ΜΕΡΟΣ Β-Ερώτημα 1

### DISCUSSION THREAD

**Discussion by iparthenios** [sdy60/syllabus#107] [Q&A] [answered]
Οι "σχετικές ερευνητικές εργασίες" αφορούν στον τίτλο του ερωτήματος "Ορισμός προδιαγραφών και κριτηρίων αξιολόγησης" ή στο Θέμα ομπρέλα της εργασίας;

**Comment by epidrome** [comment, marked as answer]
Και στα δύο δλδ στην τομή τους
